sgtm/artemis/extract_value_from_stringified_json_object.tpl

```diff
@@ -0,0 +1,122 @@
+﻿___TERMS_OF_SERVICE___
```

Comment:
I haven't seen this before. Did you tick the community tag option? Please remove this if you did. Thanks